ecommerce/backend/getflix/settings.py

```diff
@@ -48,6 +53,19 @@ def get_secret(setting, secrets=secrets):
     except KeyError:
         raise ImproperlyConfigured("Set the {} setting".format(setting))
 
+EMAIL_BACKEND = get_secret('EMAIL_BACKEND')
+MAILER_EMAIL_BACKEND = get_secret('MAILER_EMAIL_BACKEND')
+EMAIL_HOST = get_secret('EMAIL_HOST')
+EMAIL_HOST_PASSWORD = get_secret('EMAIL_HOST_PASSWORD')
+EMAIL_HOST_USER = get_secret('EMAIL_HOST_USER')
```

Comment:
These lines are testing purpose? We should delete them before deploying to master branch

Reply:
No, ENV variables are:
export EMAIL_BACKEND="django.core.mail.backends.smtp.EmailBackend"
export MAILER_EMAIL_BACKEND=$EMAIL_BACKEND
export EMAIL_HOST="smtp.gmail.com"
export EMAIL_HOST_PASSWORD="getflixecommerce"
export EMAIL_HOST_USER="getflix.e.commerce"
export DEFAULT_FROM_EMAIL=$EMAIL_HOST_USER

group should add these to env variables